schemas/bolt-inventory.json

```diff
@@ -0,0 +1,228 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Bolt Inventory",
+  "description": "Bolt Inventory inventory.yaml Schema",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "config": {
+      "$ref": "#/definitions/config"
+    },
+    "facts": {
+      "$ref": "#/definitions/facts"
+    },
+    "features": {
+      "$ref": "#/definitions/features"
+    },
+    "groups": {
```

Comment:
I'm a bit confused about why we're committing this file? It looks like the rake task generates `bolt-inventory.schema.json`.

Reply:
`bolt-inventory.json` is the base file that `bolt-inventory.schema.json` is built on. The inventory schemas is more complicated than the others to generate from scratch, since we don't have a list of available fields similar to the config class _and_ several of the fields can reference other fields. Since most of these fields realistically won't change or be added to, they're hardcoded while the inventory options are added in during the rake task.

Reply:
Renamed the file to `bolt-inventory-base.json`.

Reply:
Ugh I thought that was the case and grepping for `File.read` just failed. Gah, Monday.